sdk/src/Services/DynamoDBv2/Custom/DataModel/Configs.cs

```diff
@@ -550,6 +559,12 @@ public DynamoDBFlatConfig(DynamoDBOperationConfig operationConfig, DynamoDBConte
         // State of the operation using this config
         internal OperationState State { get; private set; }
 
+        /// <summary>
+        /// Property indicating the name of the attribute used for derived types conversion.
+        /// Default value is "$type" if not set in the config.
+        /// </summary>
+        public string DerivedTypeAttributeName { get; set; }
```

Comment:
Expand on this documentation. I assume this is the attribute that stores the metadata used by this library to figure out the type to instantiate.